### PR TITLE
Testing Your Ruby Code - Enumerable

### PR DESCRIPTION
**Pull request by bmuhamood**
- Tested linters and all working fine. 
- All provided Test cases working fine. 
- Follow Git hub flow.

### CODE REVIEW

File: spec/enumerables_spec.rb [hunk 1]
```diff
@@ -0,0 +1,196 @@
+# rubocop: disable Layout/LineLength
+
+require_relative '../lib/enumerables'
+
+describe '#my_each' do
+  it 'return enumerator if block is not given' do
+    arr = [1, 2, 3, 4]
```
Comment (expjazz) [inline spec/enumerables_spec.rb]: - [ ] You are defining this array a lot of times. For variables like this, you can use the _:let_  to define them one time at the top of the tests. This will make your code cleaner. Do that for all variables you are using multiple times.

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hey team :+1: 

You did a very good job here. Your tests are covering a lot of edge cases, and they are very well written. 
 However, we still have a couple of details to fix:

- [ ]  You have disabled the LineLength linter. I think that is not necessary. Try to fix this issue without disabling it.

- [ ] Use the _let_ keyword to define variables at the top to not repeat yourself (for this, check the inline comment below for more detail).

Other than that it is all good. Marvelous job here :1st_place_medal: 

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by bmuhamood** [comment]
Thanks a lot for the best review. 

- Enabled the LineLength linter and fixed manually. 
- Removed all duplicate Variables and created one i.e. **let** for using later so as to make code more cleaner. 
- Fixed all linters errors. 
- All test are working fine.

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello @bmuhamood :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)
